collections/Add_A/MS_Add_A_188.xml

```diff
@@ -53,122 +57,180 @@
+                     <decoNote type="colInit">Two-line initials in plain red.</decoNote>
+                  </decoDesc>
+                  <bindingDesc>
                      <binding notAfter="1700" notBefore="1600">
-                        <p>Red leather on boards, with stamped gold ornament, German, 17th century.</p>
+                        <p>17th-century German binding. Sewn on three bands laced into wood (beech?) boards, covered with red leather, the back cover formerly with corner-pieces and a large centre-piece and clasp fittings (all now missing), the front cover with simple blind fillets; the spine with lozenge-shaped gilt ornaments; the edges of the leaves gilt. Extensively repaired.</p>
                      </binding>
                   </bindingDesc>
                </physDesc>
+               <history>
+                  <origin resp="#PA">
+                     <origDate calendar="Gregorian" notAfter="1150" notBefore="1125">12th century, second quarter</origDate>
+                     <origPlace>
+                        <country key="place_7000084">German</country>, <region>West</region>
+                        <!--ORIGINAL: German, West-->
+                     </origPlace>
+                  </origin>
+                  <provenance resp="#VD">Regular canons of <orgName key="org_145" role="fmo">St Mary and St James, Lahneck</orgName> (slightly south of Koblenz), who moved in 1326 to Mayen (west of Koblenz); with 15th century inscription: <q>Notum sit omnibus presens scriptum visuris in perpetuum quod iste liber est ecclesie sancte Marie virginis sanctique Iacobi apostoli Lůnecke quem si quis furto detraxerit aut in vadio exposuerit …</q>, and in a 17th(?)-century hand, <q>1326. Hic liber cum caeteri Maioniam <note>[Mayen]</note> translatus sub Baldewino archiepiscopo.</q>(fol.167v) </provenance>
+                  <acquisition when="1879">Thomas Boone, London bookseller, with his price-code in pencil (fol. 168r, upper left corner); bought by the Bodleian on 29 October 1879 for £10. Former Bodleian shelfmarks: ‘MSS. Bodl. Addit. A. 188’ (fol. iir).</acquisition>
+               </history>
                <additional>
                   <adminInfo>
                      <recordHist>
-                        <source>Adapted (2018) from S. J. P. Van Dijk, <title>Handlist of the Latin Liturgical Manuscripts in the Bodleian Library Oxford : Vol. 1: Mass Books</title> (typescript, 1957), Pächt and Alexander, and the Summary Catalogue (1905); with description of decoration by Elizabeth Solopova, c. 2000. <listBibl>
-                              <bibl facs="aam0607.gif" type="SC">Summary Catalogue, Vol. 5, p. 572</bibl>
-                           </listBibl>
-                        </source>
+                        <source>Description by Peter Kidd (June 2021). For previous descriptions see Bibliography, below.
```

Comment:
I would keep things simple and let people find the bibliography on their own, since it's only a couple lines down.